src/styles/blocks/card-block/content-card.scss

```diff
@@ -0,0 +1,134 @@
+.content {
+  margin: 0 auto;
+  display: grid;
+  // gap: 48px;
+  column-gap: 48px;
+  row-gap: 46px;
+  max-width: 944px;
+  padding-top: 50px;
+
+  @media (max-width:488px) {
+    grid-template-columns: 1fr;
+  }
+
+  @media (min-width:488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width:768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width:1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+
+  &__card {
```

Comment:
The width of the cards is set to 200px, this might not be responsive on all devices. Consider using relative units like percentages.